paper/paper.md

```diff
@@ -0,0 +1,190 @@
+   index: 1
+ - name: Harvard Medical School
+   index: 2
+ - name: University of Pennsylvania
+   index: 3
+ - name: Loyola University
+   index: 4
+ - name: University of Syracuse
+   index: 5
+ - name: Northwestern University
+   index: 6
+ - name: Sonat Consulting
+   index: 7
+ - name: Princeton University
+   index: 8
+ - name: NTT DATA Mathematical Systems Inc
+   index: 9
+ - name: 605
+   index: 10
+ - name: Rockets of Awesome
+   index: 11
+ - name: Epoch Capital
+   index: 12
+ - name: University of Washington
+   index: 13
+ - name: IDLab-AIRO -- Ghent University -- imec
+   index: 14
+ - name: Clean Power Research
+   index: 15
+ - name: University of Trento
+   index: 16
+ - name: Columbia University
+   index: 17
+ - name: Georgia Tech
+   index: 18
+ - name: System1 Biosciences Inc
+   index: 19
+ - name: Research Institute for Agriculture, Fisheries and Food
+   index: 20
+
+date: 6 September 2019
+bibliography: paper.bib
+---
+
+# Summary
+
+[Generalized linear models](GLMs) are
+well-established tools for regression and classification and are widely
+applied across the sciences, economics, business, and finance. They are
+especially useful because their loss functions are convex, meaning they are
+easy and efficient to fit. Moreover, they are relatively easy to interpret
+because their noise terms are well defined, and the non-linearities they contain
+are point-wise.
+
+Mathematically, a GLM is defined as follows:
+
+$$\min_{\beta_0, \beta} \frac{1}{N} \sum_{i = 1}^N \mathcal{L} (y_i, \beta_0 + \beta^T x_i)
++ \lambda \mathcal{P}(\beta)$$
+
+where $\mathcal{L} (y_i, \beta_0 + \beta^T x_i)$ is the negative log-likelihood of an
+observation ($x_i$, $y_i$), and $\lambda \mathcal{P}(\cdot)$ is the penalty that regularizes the solution,
+with $\lambda$ being a constant that controls the amount of regularization.
+
+Modern datasets can contain an enormous number of predictor variables, and
+data analysis is often exploratory. Under these conditions it is critically
+important to regularize the model to avoid overfitting the data.
+Regularization works by adding penalty terms that penalize the model parameters in
+a variety of different ways. This can be used to incorporate prior knowledge 
+about the parameters in a structured form. In Pyglmnet, we offer
+users the ability to combine different types of regularization with different noise
+distributions in the GLMs.
+
+Despite the attractiveness of regularized GLMs, the available tools in
+the Python data science eco-system are highly fragmented. Specifically:
+
+-  [statsmodels] provides a wide range of link functions but no regularization.
+-  [scikit-learn] provides elastic net regularization but only limited noise distribution options.
+-  [lightning] provides elastic net and group lasso regularization, but only for
+   linear and logistic regression.
+
+[Pyglmnet] is a response to this fragmentation. The table below compares Pyglmnet with existing toolboxes.
```

Comment:
Replace "toolboxes" with "libraries"? "Toolboxes" is not often used.

Reply:
Agreed